File: Lab7_Igor_Markevych/RozetkaFrameworkTest/TestFramework/Pages/CheckoutPage.cs
using OpenQA.Selenium;
using OpenQA.Selenium.Support.PageObjects;
using TestFramework.Elements;

namespace TestFramework.Pages
{
    public class CheckoutPage : BasePage
    {
        public CheckoutPage(IWebDriver driver) : base(driver)
        { }

        [FindsBy(How = How.Id, Using = "reciever_name")]
        private IWebElement tempNameAndSurname;
        public TextField NameAndSurname => new TextField(tempNameAndSurname);

        [FindsBy(How = How.Id, Using = "reciever_phone")]
        private IWebElement tempPhone;
        public TextField Phone => new TextField(tempPhone);

        [FindsBy(How = How.Id, Using = "reciever_email")]
        private IWebElement tempEmail;
        public TextField Email => new TextField(tempEmail);

        [FindsBy(How = How.CssSelector, Using = ".btn-link.btn-link-green.check-step-btn-link.opaque>button")]
        private IWebElement tempContinueButton;
        public Button ContinueButton => new Button(tempContinueButton);

        [FindsBy(How = How.Id, Using = "make-order")]
        private IWebElement tempMakeOrderButton;
        public Button MakeOrderButton => new Button(tempMakeOrderButton);

        [FindsBy(How = How.Id, Using = "step_delivery")]
        private IWebElement tempDeliveryBlock;
        public HtmlLabel DeliveryBlock => new HtmlLabel(tempDeliveryBlock);

    }
}

File: Lab7_TrokhymovychMykola/Lab7_TrokhymovychMykola/src/main/java/PageObjects/CheckOutPage.java
package PageObjects;

import elements.TextField;
import org.openqa.selenium.WebDriver;
import org.openqa.selenium.WebElement;
import org.openqa.selenium.support.FindBy;

public class CheckOutPage extends BasePage{

    public CheckOutPage(WebDriver driver){
        super(driver);
    }

    @FindBy(id = "reciever_name")
    public TextField nameAndSurname;

    @FindBy(id = "suggest_locality")
    public TextField locality;

    @FindBy(id = "reciever_phone")
    public TextField phone;

    @FindBy(id = "reciever_email")
    public TextField email;

    @FindBy(css = ".btn-link.btn-link-green.check-step-btn-link.opaque>button")
    public WebElement buttonToContinue;

}
